check-peer-deps.js

```diff
@@ -1,14 +1,16 @@
 const { exec } = require('sb-exec');
 const semver = require('semver');
+const commandLineArgs = require('command-line-args');
+const commandLineUsage =  require('command-line-usage');
 
 // Flags
 // Enable debug output
-const DEBUG = false;
+let DEBUG;
```

Comment:
Since we are already getting an `opts` object back from `commandLineArgs`, let's just use that.